problems/28.Palindrome Number.gs

```diff
@@ -0,0 +1,31 @@
+function isPalindrome(x: number): boolean {
+    if (x < 0) {
+        return false
+    }
+
+    let niz = uNiz(x)
+    let i = 0
+    let j = niz.length - 1
+
+    while (i != j && j > i) {
+        if (niz[i] != niz[j]) {
+            return false
+        }
+        i++
+        j--
+    }
+    return true
+};
+function uNiz(x: number): number[] {
```

Comment:
how many times do I need to say write in English? It's not a problem I will do it each time :)